winit/src/application.rs

```diff
@@ -143,39 +143,6 @@ pub trait Application {
 
                 window.request_redraw();
             }
-            event::Event::RedrawRequested(_) => {
-                debug.render_started();
-
-                if let Some(new_size) = new_size.take() {
-                    let dpi = window.hidpi_factor();
-                    let (width, height) = to_physical(new_size, dpi);
-
-                    target.resize(
-                        width,
-                        height,
-                        window.hidpi_factor() as f32,
-                        &renderer,
-                    );
-
-                    size = new_size;
-                }
-
-                let new_mouse_cursor =
-                    renderer.draw(&primitive, &debug.overlay(), &mut target);
-
-                debug.render_finished();
-
-                if new_mouse_cursor != mouse_cursor {
-                    window.set_cursor_icon(conversion::mouse_cursor(
-                        new_mouse_cursor,
-                    ));
-
-                    mouse_cursor = new_mouse_cursor;
-                }
-
-                // TODO: Handle animations!
-                // Maybe we can use `ControlFlow::WaitUntil` for this.
-            }
```

Comment:
We should be able to revert this once https://github.com/rust-windowing/winit/issues/1082 is resolved.